example/CMakeLists.txt

```diff
@@ -1,23 +0,0 @@
-# Copyright (c) 2007-2014, Lloyd Hilaiel <me@lloyd.io>
```

Comment:
There is no need to touch the unused files. It will be the source of future conflicts. Please revert this part.

Reply:
Sure. I simply took a look in [the Boost package](https://github.com/hunter-packages/boost) where these kind of things were removed and did the same thing here. I will instead keep them around.

Reply:
Boost is a **very** special package :) It's not using CMake, there is no upstream Git repository (there are a bunch of separate repos). Removing was done for optimization reason:
- https://docs.hunter.sh/en/latest/faq/why-hunter-is-slow.html#use-latest